qdrant-landing/themes/qdrant-2024/layouts/partials/docs-menu.html

```diff
@@ -0,0 +1,91 @@
+{{ with (.Site.GetPage "/headless/docs-menu") }}
```

Comment:
We need to generate this menu from the list of documentation pages, not from markdown. 
Can you use the file `qdrant-landing/themes/qdrant/layouts/partials/docs_sidebar.html` as a reference to create it? You may replace the existing HTML with your own, but please reuse the Hugo code. You don't need to rename your file.
If you encounter any issues, please let me know.